editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -124,7 +124,7 @@ describe('React Render Count Tests -', () => {
 
     const renderCountAfter = renderResult.getNumberOfRenders()
     // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`464`)
+    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`488`)
```

Comment:
hmm, is this expected? does it worth understanding what rerenders on the UI?

Reply:
I've investigated this and answered in a comment above about the dom walker caching tests, as that is the cause